antora-ui-camel/src/css/vars.css

```
@@ -117,7 +117,7 @@
   /* static */
   --static-margin: 0 auto;
   /* frontpage */
-  --frontpage-max-width: calc(1140 / var(--rem-base) * 1rem);
+  --frontpage-max-width: calc(1350 / var(--rem-base) * 1rem);
```

With the Antora rebase we have 1366px as width, would be good to be consistent here.
```suggestion
  --frontpage-max-width: calc(1366 / var(--rem-base) * 1rem);
```

---

Applied the changes as per suggestions.